cmd/kspn/main.go

```diff
@@ -27,11 +27,12 @@ import (
 	"github.com/klaytn/klaytn/cmd/utils/nodecmd"
 	"github.com/klaytn/klaytn/console"
 	"github.com/klaytn/klaytn/log"
-	"github.com/klaytn/klaytn/metrics"
 	"github.com/klaytn/klaytn/metrics/prometheus"
+	"github.com/klaytn/klaytn/metrics/utils"
```

Comment:
To clarify the package name,
```suggestion
	metricutils "github.com/klaytn/klaytn/metrics/utils"
```
or
How about changing the directory name to "github.com/klaytn/klaytn/metrics/metricutils" ?

Reply:
@aidan-kwon I understand your point, however, then the directory name would seem duplicated since `metric` appears two times in the path. How do you think?

Reply:
I understand your point. Both are fine for me ;) Let's go with the current naming.